src/java/picard/vcf/filter/FilterVcf.java

```diff
@@ -20,6 +20,7 @@
  * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
  * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
  * THE SOFTWARE.
+ * 
  */
 package picard.vcf.filter;
 
```

Comment:
(Thank you for the whole review )
That's why I've implemented the javascript filter as a VariantFilter (and not VariantContextFilter ) to be used with **FilterVcf** .
For VCF, as far as I can see, there is no existing instance of CommandLineProgram that could be used. Are you OK with creating a new subclass of CommandLineProgram , say *VcfFilterVariants* ?